src/ccs/confluent.sh

```diff
@@ -993,6 +993,17 @@ list_command() {
     fi
 }
 
+# CLI-84: If using Mac, version must be >= 10.13
+validate_os_version() {
+    local OS=$(uname -s)
+    if [[ "${OS}" = "Darwin" ]]; then
```

Comment:
comparing against a potentially empty string is a bit funky. Let's do what we do elsewhere: 
`if [[ "x${OS}" = "xDarwin" ]]; then`
and be on the safe side.

Reply:
Huh, didn't think about this -- TIL... : )